test/dynamo/test_python_dispatcher.py

```diff
@@ -130,6 +135,184 @@ def fn(x, y):
         # No recompile
         self.assertEqual(counter.frame_count, 1)
 
+    def test_vmapped_autograd_function(self):
+        eager = EagerAndRecordGraphs()
+
+        class Foo(torch.autograd.Function):
+            generate_vmap_rule = True
+
+            @staticmethod
+            def forward(x):
+                return x * 2
+
+            @staticmethod
+            def setup_context(ctx, inputs, output):
+                pass
+
+            @staticmethod
+            def backward(ctx, grad):
+                return grad * 2
+
+        @torch.compile(backend=eager, fullgraph=True)
+        def fn(x):
+            return torch.vmap(Foo.apply)(x)
```

Comment:
Can you do a more complicated (non-pointwise) test case with double vmap? Maybe use double vmap on the LinearFunction?

Reply:
Will do!